src/mumble/Database.cpp

```diff
@@ -47,7 +48,11 @@ bool Database::findOrCreateDatabase() {
 	datapaths << Global::get().qdBasePath.absolutePath();
 	datapaths << QStandardPaths::writableLocation(QStandardPaths::DataLocation);
 #if defined(Q_OS_UNIX) && !defined(Q_OS_MAC)
-	datapaths << QDir::homePath() + QLatin1String("/.config/Mumble");
+	QString configDirectory = EnvUtils::getenv(QLatin1String("XDG_CONFIG_HOME"));
+	if(configDirectory.isEmpty()) {
+		configDirectory = QLatin1String("/.config/mumble");
+	}
+	datapaths << QDir::homePath() + configDirectory;
```

Comment:
`XDG_CONFIG_HOME` is likely to be an absolute path, in which case prepending the user's home directory to it will break it.

Reply:
Thanks for the review. I have made the requested changes. Please take a look and let me know. Thanks.